README.md

```diff
@@ -31,3 +31,175 @@ Config settings will be overriden by higher-priority sources.
+                        installed)
+  -h, --help            show this help message and exit
+
+Overrides:
+  Arguments that override configured values. Defaults stated are only used
+  if no value is given or configured.
+
+  --ndirs NDIRS         number of SIM directions
+  --nphases NPHASES     number of pattern phases per SIM direction
+  --nordersout NORDERSOUT
+                        number of output SIM orders (must be <= nphases//2;
+                        safe to ignore usually)
+  --angle0 ANGLE0       angle of the first SIM angle in radians
+  --ls LS               line spacing of SIM pattern in microns
+  --na NA               detection objective's numerical aperture
+  --nimm NIMM           refractive index of immersion medium
+  --wiener WIENER       Wiener constant; lower value leads to higher
+                        resolution and noise;
+  --zoomfact ZOOMFACT   lateral zoom factor in the output over the input
+                        images;
+  --zzoom ZZOOM         axial zoom factor; almost never needed
+  --background BACKGROUND
+                        camera readout background
+  --usecorr USECORR     use a flat-field correction file if provided
+  --k0angles K0ANGLES K0ANGLES K0ANGLES
+                        user these pattern vector k0 angles for all
+  --otfRA OTFRA         using rotationally averaged OTF; otherwise using
+  --otfPerAngle OTFPERANGLE
+                        using one OTF per SIM angle; otherwise one OTF is
+  --fastSI FASTSI       SIM image is organized in Z->Angle->Phase order;
+  --k0searchAll K0SEARCHALL
+                        search for k0 at all time points
+  --norescale NORESCALE
+                        no bleach correction
+  --equalizez EQUALIZEZ
+                        bleach correction for z
+  --equalizet EQUALIZET
+                        bleach correction for time
+  --dampenOrder0 DAMPENORDER0
+                        dampen order-0 in final assembly; do not use for 2D
+                        SIM; good choice for high-background images
+  --nosuppress NOSUPPRESS
+                        do not suppress DC singularity in the result (good
+                        choice for 2D/TIRF data)
+  --nokz0 NOKZ0         not using kz=0 plane of the 0th order in the final
+                        assembly (mostly for debug)
+  --gammaApo GAMMAAPO   output apodization gamma; 1.0 means triangular
+  --explodefact EXPLODEFACT
+                        artificially exploding the reciprocal-space
+  --nofilterovlps NOFILTEROVLPS
+  --deskew DESKEW       Deskew angle; if not 0.0 then perform deskewing before
+                        processing
+  --deskewshift DESKEWSHIFT
+                        If deskewed, the output image's extra shift in X
+                        (positive->left)
+  --noRecon NORECON     No reconstruction will be performed; useful when
+                        combined with "deskew":
+  --cropXY CROPXY       Crop the X-Y dimension to this number; 0 means no
```

Comment:
pixels?

Reply:
I assume so, but most of this is from the cudasirecon docs, so I don't want to make assumptions